implementation/index.md

```diff
@@ -4,17 +4,17 @@ In this part of the document we present an implementation of the requirements an
 
 As explained above, any implementation is destined to be outdated sooner or later. However, the authors of the CS IP have made their best effort to reuse already available best practices and established core standards, and to carry out intensive discussions within the digital preservation community. All of the above should guarantee that the implementation can be used with only minor updates (for example minor updates to metadata elements) throughout the next few decades.
 
-# 4.	CS IP Information Package structure
-The implementation of the conceptual model described in Requirement 3.6 is a fixed physical (folder) structure which follows exactly the conceptual structure.
+# 4.	CS IP structure
+The preferred implementation of the conceptual model described in Requirement 3.6 is a fixed physical (folder) structure which follows exactly the conceptual structure. Other ways of implementing the conceptual model should not be used and are not encouraged at the same time the CS IP dont prohibit other solutions of implementing the conceptual model.
 
```

Comment:
While the CS IP doesn't prohibited alternative implementations of the conceptual model such implementations aren't recommended.